lib/logos.js

```diff
@@ -57,7 +57,7 @@ function getShieldsIcon({ name, color }) {
 
   const { svg, base64 } = logos[key]
   const svgColor = toSvgColor(color)
-  if (svgColor) {
+  if (svgColor && (svg.match(/fill="(.+?)"/g) || []).length === 1) {
```

Comment:
Could this regex test be done at the time the logos are loaded? Also could you add a comment with the English version of that? Is it "just one fill attribute"?